napari/benchmarks/benchmark_shapes_layer.py

```diff
@@ -27,48 +28,141 @@
     from napari.benchmarks.utils import Skip
 
 
-class Shapes2DSuite:
+class BackendType(Enum):
+    bermuda = (
+        auto()
+    )  # data processing using PartSegCore, ready to switch to bermuda
```

Comment:
```suggestion
    )  # data processing using PartSegCore (https://partseg.github.io)
```

Reply:
I hope to switch to Bermuda in a few days. This is last remaining PR https://github.com/napari/bermuda/pull/30

Reply:
Gotcha let me reword the explanation. Instead of end of line comment, use comment above bermuda statement. @Czaki here's a suggestion.

`# bermuda (https://github.com/napari/bermuda) is a rust-based performance library to accelerate spatial algorithms for napari.`

Reply:
I have added partsegcore to enum. Bermuda is currently unused. I will restore partsegcore in ##7747 to allow both. Could you check now?